oauth-client-util/src/main/java/org/radarcns/oauth/OAuth2Client.java

```diff
@@ -119,8 +119,16 @@ private void getNewToken() throws TokenException {
             .build();
 
         // make the client execute the POST request
-        try (Response response = getHttpClient().newCall(request).execute()) {
-            currentToken = OAuth2AccessTokenDetails.getObject(response);
+        try {
+            Response response = getHttpClient().newCall(request).execute();
```

Comment:
`Response` implements `AutoCloseable` so we can keep using it in a try-with-resources statement.